exam001/Views/ViewController.swift

```diff
@@ -0,0 +1,554 @@
+//
+//  ViewController.swift
+//  exam001
+//
+//  Created by Adnan Al-Akhrass on 11/8/18.
+//  Copyright © 2018 Adnan Al-Akhrass. All rights reserved.
+//
+
+//
+
+import UIKit
+import Firebase
+
+enum textfieldIndex: Int {
+    case first
+    case second
+    case third
+    case fourth
+}
+
+struct Requestes {
+    var fullName: String
+    var floor: String
+    var location: String
+    var items: String
+}
+
+class ViewController: UIViewController {
+
+    // MARK: - Properties
+    private lazy var firstTextField: CustomTextField = {
+        let textField = CustomTextField(fieldPlaceHolder: viewModel.fieldsArray[textfieldIndex.first.rawValue], fieldColor: .orange, fieldBorderStyle: .line)
+        return textField
+    }()
+    
+    private lazy var secondTextField: CustomTextField! = {
+        let secondTextField = CustomTextField(fieldPlaceHolder: viewModel.fieldsArray[textfieldIndex.second.rawValue], fieldColor: .orange, fieldBorderStyle: .line)
+        return secondTextField
+    }()
+
+    private lazy var thirdTextField: CustomTextField! = {
+        let thirdTextField = CustomTextField(fieldPlaceHolder: viewModel.fieldsArray[textfieldIndex.third.rawValue], fieldColor: .orange, fieldBorderStyle: .line, xPad: 10)
+        return thirdTextField
+    }()
+    
+    private lazy var fourthTextField: CustomTextField! = {
+        let fourthTextField = CustomTextField(fieldPlaceHolder: viewModel.fieldsArray[textfieldIndex.fourth.rawValue], fieldColor: .orange, fieldBorderStyle: .line, ypad: 10)
+        return fourthTextField
+    }()
+    
+    private var topButton: SampleButton!
+    
+    private var bottomButton: SampleButton!
+    
+    private lazy var submitButton: SampleButton! = { [weak self] in
```

Comment:
only use` [weak self]` if self is being used inside the block, noting that this block has no use for self then It should be fine to remove `[weak self] in` from here.